### PR TITLE
feat: update description and user stories

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39721

### CODE REVIEW

**Review by RandellDawson — DISMISSED** [review]
LGTM

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

@moT01 was adding the `blocked` label intentional?

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/file-metadata-microservice.md [hunk 1]
```diff
@@ -8,15 +8,19 @@ forumTopicId: 301506
...
+
+When you are done, make sure a working demo of your project is hosted somewhere public. Then submit the URL to it in the `Solution Link` field. Optionally, also submit a link to your projects source code in the `GitHub Link` field.
 </section>
 
 ## Instructions
 
 <section id='instructions'>
-
+**HINT:** You can use the `multer` npm package to handle file uploading.
 </section>
```
Comment (nhcarrigan) [inline curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/file-metadata-microservice.md:24]:
```suggestion

**HINT:** You can use the `multer` npm package to handle file uploading.
</section>
```
It looks like a line-break is needed for the parser to render this markdown appropriately, actually.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/file-metadata-microservice.md (hunk 1 above)
Comment (moT01) [reply curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/file-metadata-microservice.md:24]: nice catch

**Comment by moT01** [comment]
> @moT01 was adding the blocked label intentional?

Yes, I opened the PR and then noticed that hint in the boilerplate and wanted to add it here

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Tested locally, everything renders correctly and reads clearly.

Thank you for your pull request! We look forward to additional contributions!

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**